app/Http/Controllers/DashboardAdminController.php

```diff
@@ -219,6 +219,23 @@ public function registerUser(): RedirectResponse
         // create the user
         $user = User::create($validated);
 
+        $given_file = Request::file('profile_image')[0]['file'];
```

Comment:
This can be null if no image is set. 

This results in error `Trying to access array offset on value of type null`